### PR TITLE
Update integration tests to skip medical transcriptions in govcloud regions PMV

### PR DESCRIPTION
**Pull request by usajid-aws**
**Issue #:**

**Description of changes:**
GovCloud regions currently don't support medical transcriptions. This PR skips medical transcription tests in govcloud regions. 
**Testing:**
local testing. 
*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
no demo app changes. 

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
yes
2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?

no
3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
no

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Chitz — APPROVED** [review]

File: integration/js/steps/StartMeetingTranscriptionStep.js [hunk 1]
```diff
@@ -29,7 +29,10 @@ class StartMeetingTranscriptionStep extends AppTestStep {
     if (!this.useMedical) {
       await this.page.clickTranscribeEngineOption();
     } else {
-      await this.page.clickTranscribeMedicalEngineOption();
+      // Skip medical transcription test in govCloud regions.
+      if(process.env.REGION !== 'us-gov-east-1' && process.env.REGION !== 'us-gov-west-1') {
```
[the remark this answers is not in the document]
Comment (noahmako) [reply integration/js/steps/StartMeetingTranscriptionStep.js:33]: This is a weird place to gate the logic. We still start transcription but don't select medical first, which means we will test normal transcription twice, right?

[the remark this answers is not in the document]
Comment (usajid-aws) [reply integration/js/steps/StartMeetingTranscriptionStep.js]: logic moved to TranscriptionTest.js.

**Review by noahmako — APPROVED** [review]

File: integration/js/TranscriptionTest.js [hunk 2]
```diff
@@ -65,7 +65,9 @@ class TranscriptionTest extends SdkBaseTest {
     await this.runTranscriptionTest(session, testWindow1, testWindow2, false, testWindow1, testWindow2, testWindow1, attendeeId1, compareFn);
 
     // Transcribe Medical
-    await this.runTranscriptionTest(session, testWindow1, testWindow2, true, testWindow2, testWindow1, testWindow2, attendeeId2, compareFn);
+    if (process.env.REGION !== 'us-gov-east-1' && process.env.REGION !== 'us-gov-west-1') {
```
Comment (devalevenkatesh) [inline integration/js/TranscriptionTest.js:68]: Can the payload take the non-supported regions in config itself?
https://github.com/aws/amazon-chime-sdk-js/blob/main/integration/configs/integrationTest/transcription_test.config.json#L12-L16

Then have a separate function to compare and support both the payload as well as `process.env.REGION`. This separate function decides whether to run transcription test or not.

Comment (devalevenkatesh) [reply integration/js/TranscriptionTest.js:68]: NVM, this is only for medical transcription so should be fine.

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**